download_java_clients.sh

```diff
@@ -4,10 +4,14 @@ function install_java_client(){
     rm -rf ./build
     rm -rf ./cli-java
     mkdir -p build/java_clients
-    wget https://github.com/rh-messaging/cli-java/releases/download/v2.4.0er2/cli-activemq-1.2.2-SNAPSHOT-5.11.0.redhat-630377.jar -O build/java_clients/cli-activemq.jar
-    wget https://github.com/rh-messaging/cli-java/releases/download/v2.4.0er2/cli-artemis-jms-1.2.2-SNAPSHOT-2.7.0.redhat-00056.jar -O build/java_clients/cli-artemis-jms.jar
-    wget https://github.com/rh-messaging/cli-java/releases/download/v2.4.0er2/cli-qpid-jms-1.2.2-SNAPSHOT-0.42.0.redhat-00001.jar -O build/java_clients/cli-qpid-jms.jar
-    wget https://github.com/rh-messaging/cli-java/releases/download/v2.4.0er2/cli-paho-java-1.2.2-SNAPSHOT-RELEASE.jar -O build/java_clients/cli-paho-java.jar
+    git clone https://github.com/rh-messaging/cli-java
+    cd cli-java
+    mvn clean package
+    cd ..
+    cp cli-java/cli-activemq/target/cli-activemq-1.2.2-SNAPSHOT-5.15.10.jar build/java_clients/cli-activemq.jar
```

Comment:
That will not work becasue next release you have to again rewrite all versions, so you should use some pattern to copy it.